src/unstructured_client/general.py

```diff
@@ -12,7 +14,7 @@ def __init__(self, sdk_config: SDKConfiguration) -> None:
         self.sdk_configuration = sdk_config
         
     
-    
+    @suggest_defining_url_if_401 # human code
```

Comment:
This isn't being linted, I can tell because there's only one space in front of the `#` for the comment. Isn't there a lint step in the CI process?

Reply:
Not on this repo. I am manually blackening the files I missed this one I guess.

Reply:
Actually, the reason I didn't lint this one is because it's one of Speakeasy's and any linting on this file would be overwritten the next time Speakeasy made a PR.